docs/resources/Webhook.md

```diff
@@ -8,16 +8,18 @@ Used to represent a webhook.
-| user?      | [user](#DOCS_RESOURCES_USER/user-object) object | the user this webhook was created by (not returned when getting a webhook with its token) |
-| name       | ?string                                         | the default name of the webhook                                                           |
-| avatar     | ?string                                         | the default avatar of the webhook                                                         |
-| token?     | string                                          | the secure token of the webhook (returned for Incoming Webhooks)                          |
+| Field           | Type                                                             | Description                                                                                      |
+| --------------- | ---------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ |
+| id              | snowflake                                                        | the id of the webhook                                                                            |
+| type            | integer                                                          | the [type](#DOCS_RESOURCES_WEBHOOK/webhook-object-webhook-types) of the webhook                  |
+| guild_id?       | snowflake                                                        | the guild id this webhook is for                                                                 |
+| channel_id      | snowflake                                                        | the channel id this webhook is for                                                               |
+| user?           | [user](#DOCS_RESOURCES_USER/user-object) object                  | the user this webhook was created by (not returned when getting a webhook with its token)        |
+| name            | ?string                                                          | the default name of the webhook                                                                  |
+| avatar          | ?string                                                          | the default avatar of the webhook                                                                |
+| token?          | string                                                           | the secure token of the webhook (returned for Incoming Webhooks)                                 |
+| source_guild?   | partial [guild](#DOCS_RESOURCES_GUILD/guild-object) object       | the guild of the channel that this webhook is following (returned for Channel Follower Webhooks) |
+| source_channel? | partial [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object | the channel that this webhook is following (returned for Channel Follower Webhooks)              |
```

Comment:
I think this could be more explicit. The returned channel/guild is only a name/id respectively so partial is a bit of an understatement.

Reply:
This is handled for "partial emojis" by linking to an example. I don't think it's the best solution, but it is an idea. See https://discord.com/developers/docs/topics/gateway#message-reaction-add-message-reaction-add-event-fields

Reply:
What about a channel follower webhook example? Or would it be better to explicitly state that `source_guild` only has `id`, `name`, and `icon` and that `source_channel` only has `id` and `name`?